src/tilesource.js

```diff
@@ -709,6 +709,166 @@ $.TileSource.prototype = {
                y >= 0 &&
                x < numTiles.x &&
                y < numTiles.y;
+    },
+
+    /**
+     * Decide whether tiles have transparency: this is crucial for
+     * @return {boolean} true if the image has transparency
+     */
+    hasTransparency: function(context2D, url, ajaxHeaders, post) {
+        return !!context2D || url.match('.png');
+    },
+
+    /**
+     * Download tile data
+     * @param {object} context job context that you have to call finish(...) on. It also contains abort(...) function
+     *   that can be called to abort the job.
+     * @param {String} [context.src] - URL of image to download.
+     * @param {String} [context.loadWithAjax] - Whether to load this image with AJAX.
+     * @param {String} [context.ajaxHeaders] - Headers to add to the image request if using AJAX.
+     * @param {String} [context.crossOriginPolicy] - CORS policy to use for downloads
+     * @param {String} [context.postData] - HTTP POST data (usually but not necessarily in k=v&k2=v2... form,
+     *      see TileSrouce::getPostData) or null
+     * @param {Function} [context.callback] - Called once image has been downloaded.
```

Comment:
No, I updated the docs. It is called by finish. Finish basically receives t/f success and possibly error message, calls the callback and calls also `doanloadTileFinish`. The callback could be possibly left out from the documentation since it is sort-of privately used. Maybe underscore?

Reply:
Yeah, I suppose if it's not really being used at this level, it should be documented as such. Probably good to mark it as private, rather than leaving it out of the documentation entirely.